packages/assets-controllers/src/NftController.test.ts

```diff
@@ -149,6 +184,24 @@ function setupController({
     getNetworkClientById,
   );
 
+  const getInternalAccountMock = jest
+    .fn()
+    .mockReturnValue(defaultSelectedAccount ?? OWNER_ACCOUNT);
+
+  messenger.registerActionHandler(
+    'AccountsController:getAccount',
+    getInternalAccountMock,
+  );
+
+  const getSelectedAccountMock = jest
```

Comment:
Nit: We could have used `mockGetSelectedAccount` like in other PRs (if we do so, we would need to rename all of the in this file, but this is fine, since this is something new)